syft/serde.py

```diff
@@ -1238,6 +1244,16 @@ def _detail_object_wrapper(
     return obj_wrapper
 
 
+def _simplify_exception(e):
+    error_message = sys.exc_info()
+    return pickle.dumps(error_message)
```

Comment:
I'm somewhat opposed to using pickle for this as pickle is not supported on other systems (Ie js) further we are doing nothing with the error passed to this function. Is there a better way to do this? For other errors that we have serialized in the past, we built a manual serializer

Reply:
You're right, the good thing with this is that you get the same stacktrace with code from remote workers included, which is a bit amazing. But in this very case the exception ResponseSignatureError is caught so you don't get to see how cool the stacktrace is.

I can try an alternative solution